src/group.coffee

```diff
@@ -239,3 +240,22 @@ module.exports = (robot) ->
       res.send "#{name} is in #{group.membership(name).join ", "}."
     else
       res.send "#{name} is not in any groups!"
+
+  robot.respond ///group\s+add\s+(#{IDENTIFIER})\s+(&?#{IDENTIFIER}(?:\s+&?#{IDENTIFIER})*)///, (res) ->
+    g = res.match[1]
+    names = res.match[2]
+    names = names.split /\s+/
+    if not group.exists g
+      res.send "Group #{g} does not exist!"
+      return
+    response = []
+    for n in names
+      for k, u of robot.brain.data.users
```

Comment:
Maybe add a case that gives an error message when the name you put in doesn't match any of the names?